packages/expo-ads-admob/unimodule.json

```diff
@@ -0,0 +1,4 @@
+{
+  "name": "expo-ads-admob",
+  "platforms": ["ios", "android"]
```

Comment:
Is it possible to add web/would we need to?

Reply:
Yeah, we could add `web` here, but I'm not sure if this is needed since auto-installation won't be used on web platform. But maybe we can add it just for information purposes? What do you think?

Reply:
Maybe if use this for bundling, native.directory, or flutter in the future. Otherwise ehh doesn't seem super needed right now.